codalab/server/bundle_manager.py

```diff
@@ -632,7 +632,7 @@ def _schedule_run_bundles(self):
         READY / FAILED, no worker_run DB entry:
             Finished.
         """
-        workers = WorkerInfoAccessor(self._worker_model.get_workers())
+        workers = WorkerInfoAccessor(self._worker_model, WORKER_TIMEOUT_SECONDS - 5)
```

Comment:
It's safer to renew slightly more frequently than the timeout to avoid a case where we get the worker states from this object say 59 seconds after it was last updated, but by the time we check the timestamps, it's already been 60 seconds. I can set it to be 1 just to show that it's a symbolic safety measure.